src/web/services/feedback-responses.service.ts

```diff
@@ -122,6 +126,35 @@ export class FeedbackResponsesService {
     }
   }
 
+  /**
+   * Determines whether responses should be displayed based on the selected section.
+   */
+  isFeedbackResponsesDisplayedOnSection(section: string, sectionType: InstructorSessionResultSectionType,
+      response: ResponseOutput): boolean {
+
+    let isDisplayed: boolean = true;
+
+    if (section) {
+      switch (sectionType) {
+        case InstructorSessionResultSectionType.EITHER:
+          isDisplayed = response.giverSection === section || response.recipientSection === section;
+          break;
+        case InstructorSessionResultSectionType.GIVER:
+          isDisplayed = response.giverSection === section;
+          break;
+        case InstructorSessionResultSectionType.EVALUEE:
+          isDisplayed = response.recipientSection === section;
+          break;
+        case InstructorSessionResultSectionType.BOTH:
+          isDisplayed = response.giverSection === section && response.recipientSection === section;
+          break;
+        default:
```

Comment:
```suggestion
```
Could do without the empty `default` imo, clean the code up a little since you're doing the refactoring haha

Reply:
There are lint restrictions that prevent from doing so.

`ERROR: /Users/xinjiezhuang/Desktop/TEAMMATES/teammates/src/web/services/feedback-responses.service.ts:138:7 - Switch statement should include a 'default' case`

`ERROR: /Users/xinjiezhuang/Desktop/TEAMMATES/teammates/src/web/services/feedback-responses.service.ts:150:11 - Final clause in 'switch' statement should not end with 'break;'.`